specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/WebApps.json

```diff
@@ -28742,6 +28742,28 @@
               "description": "HttpsOnly: configures a web site to accept only https requests. Issues redirect for\nhttp requests",
               "type": "boolean"
             },
+            "minTlsCipherSuite": {
```

Comment:
Noticed you added it to the SitePatchResource, might have to go in the SiteConfig resource in the CommonDefinitions file instead

Reply:
I moved it to CommonDefinitions. But should it be in both? What is the SitePatchResource for?